media/css/mozorg/developer/index.scss

```diff
@@ -0,0 +1,363 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../pebbles/components/newsletter';
+@import '../../hubs/mixins';
+@import '../../hubs/masthead';
+@import '../../hubs/sections';
+@import '../../hubs/sub-nav';
+@import '../../hubs/buttons';
+@import '../../hubs/cards-block';
+@import '../../hubs/card-image-large';
+@import '../../hubs/common';
+@import '../../hubs/news-feed';
+@import 'hub/newsletter';
+
+main {
+    @include font-size(20px);
+    @include zilla-slab;
+
+    .content {
+        @include horizontal-rule-before($start-color: #8edc34, $end-color: #82d1f5); // light green to light blue
+    }
+}
+
+// Hero promo doesn't get a gradient rule
+.devtools .content:before {
+    content: none;
+}
+
+/* -------------------------------------------------------------------------- */
+// Page header
+
+.header-intro {
+    h1 {
+        color: #fff;
+        line-height: 1.3;
+
+        span {
+            @include box-decoration-break(clone);
+            background: #000;
+        }
+    }
+
+    .tagline {
+        @include font-size-level3;
+        color: $color-text-secondary;
+        font-weight: bold;
+        line-height: 1.2;
+        margin-bottom: 0;
+    }
+
+    .content:before {
+        height: 10px;
+    }
+
+    @media #{$mq-tablet} {
+
+        .header-container {
+            @include clearfix;
+        }
+
+        .header-content {
+            @include span(6);
+            padding-left: 0;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        h1 {
+            @include font-size(72px);
+        }
+
+        .tagline {
+            @include font-size(32px);
+        }
+
+        .header-content {
+            @include span(8);
+            padding-left: 0;
+        }
+    }
+
+    @media #{$mq-desktop-wide} {
+        h1 {
+            @include font-size(95px);
+        }
+
+        .tagline {
+            @include font-size(48px);
+        }
+
+        .header-content {
+            @include span(9);
+            padding-left: 0;
+        }
+    }
+}
+
+html[dir="rtl"] {
+    .header-intro .header-content {
+        float: right;
+        padding: 0 0 0 10px;
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Cards
+
+.card-group {
```

Comment:
Absolutely. I think I've added some fresh debt here and we still need to refactor other pages. I started by reusing the features list from your pages but of course these items aren't a list of features so that felt wrong. I went for generic "cards" and a generic "group of cards" with some minor variations. Maybe this can become the standard pattern and we'll update other pages to follow?